config/configclientauth/README.md

```diff
@@ -0,0 +1,54 @@
+# Client Authorization Settings
+
+This module provides HTTP/GRPC client types to be configured to perform authorization for requests using the following 
```

Comment:
```suggestion
This module provides helpers for HTTP/gRPC exporters, allowing them to be configured to perform authentication of outgoing requests.
```

Reply:
Hmm, makes me think - The ultimate goal in both OAuth2 client credentials and "Bearer Token" way is to get a token for authorization to access resources on the server - not sure how to go about it.

Reply:
```
# AuthX Settings for Exporters

This module provides HTTP/GRPC client types to be configured to perform authorization/authentication for requests using the following 
```
would something like this work?

Reply:
I would still use "HTTP/gRPC exporters" instead of client types, to make it extra clear that this is affecting exporters, not extensions nor receivers. But despite the name of the HTTP header being "authorization", sending credentials is only the authentication part of the story: it's telling the server who the client is (username), and giving some proof of it (secret/password). Authorization (is the user authorizes to perform the requested operation?) is solely done on the server-side.

Reply:
@jpkrohling got it! I have modified the README per your original suggestion. Please let me know - Thanks for the review!